EquipmentManager/Pages/Home.razor

```diff
@@ -51,12 +50,11 @@
         var hc = new HttpClient();
         // localhost:5284
 
-        var builder = WebAssemblyHostBuilder.CreateDefault();
-        var hostname = builder.Configuration["HostName"];
+        var ItemGeneratorUrl = Configuration["ItemGeneratorUrl"];
 
-        if (string.IsNullOrEmpty(hostname)) hostname = "http://eiga.pollub.net/";
+        if (string.IsNullOrEmpty(ItemGeneratorUrl)) ItemGeneratorUrl = "http://eiga.pollub.net/item";
```

Comment:
to jest okej, ale akurat jako ciekawostka można to uprościć do formy

```cs
var itemGeneratorUrl  = Configuration["ItemGeneratorUrl"] ?? "http://eiga.pollub.net/item";
```

korzystając z [null-coalescing operatora](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator)

Reply:
próbowaliśmy tak zrobić, ale jak jest null w appsettings.json, to 
`Configuration["ItemGeneratorUrl"] ` zwraca pusty string)

Reply:
aaaa, jak tak to spoko